### PR TITLE
New increment without separators

### PR DESCRIPTION
**Pull request by xconverge**
Yay! We love PRs! 🎊

Please include a description of your change and ensure:

- [X] Commit message has a short title & issue references
- [X] Each commit does a logical chunk of work.
- [X] It builds and tests pass (e.g `gulp`)

More info can be found on our [contribution guide](https://github.com/VSCodeVim/Vim/blob/master/.github/CONTRIBUTING.md).

@sectioneight @johnfn took another stab at this one, last attempt was https://github.com/VSCodeVim/Vim/pull/661

### CODE REVIEW

File: src/number/numericString.ts [hunk 1]
```diff
@@ -15,18 +17,53 @@ export class NumericString {
...
+
+      // Regex to find any trailing characters after the number
+      let findSuffix = /[^\d]*$/i;
+
+      let newPrefix = prefix;
+      let newSuffix = "";
+      let newNum = input;
+
+      // Only use this section if this is a number surrounded by letters
+      if (findNondigits.exec(input) !== null && NumericString.matchings[NumericString.matchings.length - 1].regex === regex) {
```
Comment (xconverge) [inline src/number/numericString.ts]: Would like feedback on how I can get an ID to only run this for 1 of the 4 regex situations

Comment (johnfn) [inline src/number/numericString.ts]: What do you mean? I don't understand.

Comment (xconverge) [inline src/number/numericString.ts]: Right here I am only doing this logic if it is the last regex in the list (none of the others matched, so the last one is a catchall for anything that contains any number) It seems like a hack to depend on the last regex in the list to be this catchall...

**Comment by xconverge** [comment]
This should be good to go unless someone has some more feedback, I tested it a bit more and am happy enough with it

**Comment by rebornix** [comment]
LGTM.

**Pull request merged**